contracts/8.17/stablecoin-core/LiquidationEngine.sol

```diff
@@ -64,7 +64,7 @@ contract LiquidationEngine is PausableUpgradeable, ReentrancyGuardUpgradeable, I
   }
 
   // --- Init ---
-  function initialize(address _bookKeeper, address _systemDebtEngine) external initializer {
+  function initialize(address _bookKeeper, address _systemDebtEngine, address _priceOracle) external initializer {
```

Comment:
Motivation is that address _priceOracle is used in many functions anyway. It makes more sense to configure it from the beginning.